__docs__/wonder-blocks-modal/flexible-dialog.stories.tsx

```diff
@@ -235,19 +235,19 @@ export const WithAriaLabel: StoryComponentType = {
 
 /**
  *
- * A FlexibleDialog can have an aria-label as its accessible name.
+ * A FlexibleDialog can derive its accessible name from aria-labelledby.
  */
 export const WithAriaLabelledby: StoryComponentType = {
     render: () => (
         <View style={styles.previewSizer}>
             <View style={styles.modalPositioner}>
                 <FlexibleDialog
                     aria-labelledby="main-heading"
+                    title={
```

Comment:
Does this mean the heading should always be in the `title` prop? Are there use cases that consumers would need where it is part of the `content`?